frameworks/keyed/ember/src/components/the-table.gjs

```diff
@@ -0,0 +1,27 @@
+import Component from '@glimmer/component';
+import { fn } from '@ember/helper';
+import { getOwner } from '@ember/owner';
+
+export class TheTable extends Component {
+  state = getOwner(this).lookup('service:state');
+
+  <template>
+    <table class="table table-hover table-striped test-data">
+      <tbody>
+        {{#each this.state.data as |row|}}
+          <tr class={{if (this.state.isSelected row) "danger" ""}}><td
+              class="col-md-1"
+            >{{row.id}}</td><td class="col-md-4"><a
+                onclick={{fn this.state.select row}}
+              >{{row.label.current}}</a></td><td class="col-md-1"><a
+                onclick={{fn this.state.remove row}}
```

Comment:
remove don't really working 
<img width="591" alt="image" src="https://github.com/user-attachments/assets/c74e1a33-265d-48ab-8152-b1cbd4bd6777" />

Reply:
I wonder if I misread the bench results. HMM

thanks for catching this

Reply:
fixed